07LessonSeven/homework/css/style.css

```diff
@@ -0,0 +1,62 @@
+body{
+  font-family: sans-serif;
+  margin: auto;
+}
+p{
+  font-size: 14px;
+}
+header h1{
+  margin-left: 150px;
+  margin-bottom: 10px;
+  margin-top: 10px;
+}
+nav{
+  background-color: #313131;
+  height: 70px;
+  width: 100%;
+  line-height: 70px;
+}
+header nav a:first-child{
+  margin-left: 150px;
+}
+header nav a {
+  color: #fff;
+  text-decoration: none;
+}
+header nav a:nth-child(2){
+  margin-left: 20px;
+  margin-right: 20px;
+}
+div{
+  width: 80%;
+  margin: auto;
+  color: #fff;
+}
+section{
+  width: 50%;
+  height: 250px;
+  background-color: #55a747;
+  float: left;
+  margin-top: 15px;
+}
+aside{
+  width: 50%;
+  height: 250px;
+  background-color: #313131;
+  float: left;
+  margin-top: 15px;
+}
+article{
+  width:80%;
+  height: 80%;
+  margin-left: auto;
+  margin-right: auto;
+  margin-top: 70px;
+}
+article a{
+  color: white;
+  text-decoration: none;
+  border: 2px solid #fff;
+  padding: 10px;
+  margin-left: 285px;
```

Comment:
because of this margin the link it's not fitting right and breaks into two lines 
check it out [here](https://trevorlujan123.github.io/intro-workbook/07LessonSeven/homework/)